src/Symfony/Bridge/Twig/Extension/SecurityExtension.php

```diff
@@ -55,6 +55,16 @@ public function isGranted(mixed $role, mixed $object = null, ?string $field = nu
         }
     }
 
+    public function accessDecision(mixed $role, mixed $object = null, ?string $field = null, ?AccessDecision $accessDecision = null): AccessDecision
+    {
+        $accessDecision = $accessDecision ?? new AccessDecision();
```

Comment:
not there's a point for templates to pass a previous access decision:
```suggestion
    public function accessDecision(mixed $role, mixed $object = null, ?string $field = null): AccessDecision
    {
        $accessDecision = new AccessDecision();
```